MAVProxy/mavproxy.py

```diff
@@ -269,15 +269,18 @@ def cmd_watch(args):
 
 def generate_kwargs(args):
     kwargs = {}
-    module_components = args.split(":{", 1)
+    module_components = args.split(":", 1)
     module_name = module_components[0]
-    if (len(module_components) == 2 and module_components[1].endswith("}")):
```

Comment:
Ideally I would like to see this section of code remain as is and handle the comma separated module list prior to `modlist = module.split(',')`